src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -1206,6 +1206,14 @@ private TileEntity getTile(ICraftingMedium craftingProvider) {
 
     }
 
+    public int getRemainingOperations() {
+        if (this.isComplete) {
+            return this.accelerator + 1;
```

Comment:
What does it mean? (I mean - why remaining operations of the finished process is not 0?)

Reply:
in the ``CraftingCPUCluster::updateCraftingLogic()`` 
``        if (this.waiting || this.tasks.isEmpty()) // nothing to do here...
        {
            return;
        }

        this.remainingOperations = this.accelerator + 1 - (this.usedOps[0] + this.usedOps[1] + this.usedOps[2]);
        final int started = this.remainingOperations;

        if (this.remainingOperations > 0) {
            do {
                this.somethingChanged = false;
                this.executeCrafting(eg, cc);
            } while (this.somethingChanged && this.remainingOperations > 0);
        }
        this.usedOps[2] = this.usedOps[1];
        this.usedOps[1] = this.usedOps[0];
        this.usedOps[0] = started - this.remainingOperations;

        if (this.remainingOperations > 0 && !this.somethingChanged) {
            this.waiting = true;
        }
``
That's why i plus 1